multigeiger/multigeiger.ino

```diff
@@ -200,7 +200,10 @@ void read_THP(unsigned long current_ms,
 }
 
 void transmit(unsigned long current_ms, unsigned long current_counts, unsigned long gm_count_timestamp, unsigned long current_hv_pulses,
-              bool have_thp, float temperature, float humidity, float pressure) {
+              bool have_thp, float temperature, float humidity, float pressure, bool wificonnected) {
+  if( !wificonnected) {
+    return;
```

Comment:
guess this kills a lora device that currently has no wifi connection (which is kind of expected).

Reply:
so there should be a "if wificonnected" around the transmission stuff needing wifi and another "if lora available" around the transmission stuff needing lora.

Reply:
if wificonnected -> OK
But we don't have a 'if lora available'. If we have lora hardware, we have lora available. If we cannot join, we get an error but we have to try the join in any way.
So if 'sendToLora' is true, we always (try to) send.